sale_order_amount_to_invoice/models/sale_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    amount_to_invoice = fields.Monetary(
```

Comment:
Should be `untaxed_amount_to_invoice` because it's the sum of  `untaxed_amount_to_invoice`  of the SO lines

Reply:
Thanks, I'll fix it)